src/argparser.py

```diff
@@ -324,6 +324,15 @@ def validate_block(value):
         help="\t\t\tSet the format of output file of rocprof.",
     )
 
+    profile_group.add_argument(
+        "--pc-sampling-interval",
+        required=False,
+        metavar="",
+        dest="pc_sampling_interval",
+        default=1,
+        help="\t\t\tSet the interval of pc sampling in microsecond.",
```

Comment:
```suggestion
        help="\t\t\tSet the interval of pc sampling in microsecond (DEFAULT: 1).",
```
In other options, we state the default value in help print out

Reply:
Fixed with 22f00b27. I mixed C++ parser and python:)